src/mam/mod_mam_muc_rdbms_arch.erl

```diff
@@ -86,6 +87,7 @@ start_hooks(Host, _Opts) ->
     ejabberd_hooks:add(mam_muc_archive_size, Host, ?MODULE, archive_size, 50),
     ejabberd_hooks:add(mam_muc_lookup_messages, Host, ?MODULE, lookup_messages, 50),
     ejabberd_hooks:add(mam_muc_remove_archive, Host, ?MODULE, remove_archive, 50),
+    ejabberd_hooks:add(remove_domain, Host, ?MODULE, remove_domain, 50),
```

Comment:
please, make a hooks/0 function that returns a list of hooks and use ejabbed_hooks:add/1 and ejabberd_hooks:delete/1 functions to start/stop module